wagtail_draftail_anchors/static_src/wagtaildraftailanchors/js/wagtail_draftail_anchor.js

```diff
@@ -107,6 +107,31 @@ window.draftail.registerPlugin({
   decorator: AnchorIdentifier,
 });
 
+const CopyAnchorButton = ({identifier}) => {
+  const [didCopy, setDidCopy] = React.useState(false);
+
+  const copyText = (event) => {
+    // Prevent the button click event from submitting the page form
+    event.preventDefault();
+    navigator.clipboard.writeText(identifier);
+    setDidCopy(true);
+  }
+
+  const classes = `button button-small ${didCopy ? "button-secondary" : ""}`;
```

Comment:
Will do full review later, but at a glance, colour contrast looks like it could be inaccessible once it's clicked and switches to the secondary option.

Reply:
I think you're right. I used Wagtail's `button-secondary` class originally, but it looks like that inherits the background colour, and doesn't work well here. Changed so the classes remain the same after copying